docs/avatar.html

```diff
@@ -76,6 +79,9 @@ <h5 class="m-none">Components</h5>
           <li>
             <a href="/docs/navigation.html" class="navlink">Navigation</a>
           </li>
+          <li>
+            <a href="/docs/input.html" class="navlink">Input</a>
+          </li>
         </nav>
       </aside>
       <section id="avatar" class="mb-2 p-1 flex-1">
```

Comment:
Just a code suggestion of you a recursively using pb-1 flex utility class create classname and inherite those properties this will save time to add each class everytime